### PR TITLE
Milestone: Add Remove ToDoList

### PR DESCRIPTION
**Pull request by Gleennkar**
## Created the Add Remove ToDoList
- Set up Webpack
- Created another index.js file to give the various functionality
- Export files from the index.js to the addremove.js file
- Check for linter errors

### CODE REVIEW

File: src/addRemoveToDo.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+if (storedList === null) {
+  storedList = [];
+}
+
+export const add = (description) => {
+  const todolist = [...storedList];
+  todolist.push({
+    index: Date.now(),
+    description,
+    completed: false,
```
Comment (Danieldotcomcoder) [inline src/addRemoveToDo.js:16]: when I mark a task as completed , the value of completed of a given task in the local storage should be updated to true , In your case its not updating, Kindly fix this issue 👍 👍

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @Gleennkar  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/update.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  event.target.nextElementSibling.classList.toggle('completed');
+  const status = JSON.parse(localStorage.getItem('tasks'));
+  if (status[event.target.parentNode.parentNode.id].completed === true) {
+    status[event.target.parentNode.parentNode.id].completed = false;
+  } else {
+    status[event.target.parentNode.parentNode.id].completed = true;
+  }
+  localStorage.setItem('tasks', JSON.stringify(status));
+}
+export default updateStatus;
```
Comment (ahmetbozaci) [inline src/update.js:11]: - [ ] As you can see status still has not changed in the local storage. Kindly fix it. 

note: I refresh the storage.
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️ 


@Gleennkar  ✋🏼 
#### Great work on making the changes requested by a previous reviewer 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
___

### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
@Gleennkar 
⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 

- Your commit messages should always start with capital letters and they should be an imperative tense. I see some of them are not like that. Instead of writing `Added style or Created the file` you should write `Add style or Create the file`
- ***I write it as optional. But this is a [general requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=The%20commit%20message%20should%20be%20written%20in%20the%20imperative%20tense%20and%20capitalized) for every project. So you should pay attention to this in your next projects.***
- You can change them by using `git-rebase`. If you want to know how to change ı leave a few resources.
- [Video - How to change commit messages](https://www.loom.com/share/00771f369b7b47fc968c019ecf0d3ba6)
- [Article - How to create good commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)

⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by bimbolabuari — COMMENTED** [review]
Hello @Gleennkar  👋, 

_This is an additional review as a result of the second opinion request._ 
- The requested change is for the interactive milestone, For this milestone:
   - By default, your new tasks should have the property completed set to false, which is already implemented 

- For the interactive milestone:
     - You should have one new JavaScript files imported as a module:
       - it should contain methods related to the status updates (completed: true / false).
     - It has event listener on the checkbox (change).

As this was the only reason that your project was not approved it will be approved as soon as possible. 
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :) 
Keep rocking 🚀!

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @Gleennkar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

- [ ] Please know that the property index should be set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5), kindly fix that it is meant to be a required change but it wasn't asked before by previous reviewers.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**